recipe/meta.yaml

```diff
@@ -19,16 +19,18 @@ requirements:
         - hdf5
         - libnetcdf
         - udunits2
-        - krb5
         - expat
+        - krb5
+        - esmf
```

Comment:
> We really need a nicer way of handling optional dependencies (nco still works without esmf even after this change, right?)

Indeed. The problem IMO happens in two places.

- conda does not have the concept of optional dependencies (like `pip install ipython[notebook]`);
- `nco` needs a special flag at build time to find ESMF.

We can get away with the former, but I cannot see how to circumvent the latter without building twice and renaming the package.

PS: Right now ESMF support is not working! This PR is just a test and I won't merge it.